editor-plugins.md

```diff
@@ -82,6 +82,193 @@ The diagnostics show you the syntax and semantic errors in the source code. Vari
 
 #### Debugging
 
+The VS Code Ballerina extension allows you to either run your Ballerina program (without debugging) or debug them easily by launching its debugger.
+
+>**Info:** For more information on debugging your code using VS Code, go to the [VS Code Documentation](https://code.visualstudio.com/docs/editor/debugging).
```

Comment:
Since we are not adding `debugging features` section here, I'm keeping this section as it is. WDYT?

Reply:
@praveennadarajah Yeah but I'm just afraid that the having a reference to the vscode guide at the top would be misleading since a user might go there and refer it instead of going through our Ballerina specific guide. IMO the only context that the user might want to refer the vscode guide is, when trying to use the debugger features (not at the introduction)

Reply:
As per the offline discussion, fixed with https://github.com/mohanvive/editor-pugins/pull/1/commits/6aef9a216f40f08db8f00b9ed78ef43d681ebd72